Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSupportCards.java

```diff
@@ -523,30 +524,30 @@ public class ScryfallImageSupportCards {
             add("DMR"); // Dominaria Remastered
             add("ONE"); // Phyrexia: All Will Be One
             add("ONC"); // Phyrexia: All Will Be One Commander
-            add("PL23"); // Year of the Rabbit 2023
+            add("PL23"); // Year Of The Rabbit 2023
             add("SLP"); // Secret Lair Showdown
-            add("MOM"); // March of the Machine
-            add("MOC"); // March of the Machine Commander
-            add("MAT"); // March of the Machine: The Aftermath
+            add("MOM"); // March Of The Machine
+            add("MOC"); // March Of The Machine Commander
+            add("MAT"); // March Of The Machine: The Aftermath
             add("MUL"); // Multiverse Legends
             add("30A"); // Thirtieth Anniversary Edition
-            add("LTR"); // The Lord of the Rings: Tales of Middle-Earth
-            add("LTC"); // Tales of Middle-Earth Commander
+            add("LTR"); // The Lord Of The Rings: Tales Of Middle-Earth
+            add("LTC"); // Tales Of Middle-Earth Commander
             add("CMM"); // Commander Masters
             add("WHO"); // Doctor Who
-            add("WOE"); // Wilds of Eldraine
-            add("WOT"); // Wilds of Eldraine: Enchanting Tales
-            add("WOC"); // Wilds of Eldraine Commander
-            add("LCI"); // Lost Caverns of Ixalan
-            add("LCC"); // Lost Caverns of Ixalan Commander
+            add("WOE"); // Wilds Of Eldraine
```

Comment:
Wrong naming, must use official/scryfall naming (lower case).

Reply:
Restored comments to match scryfall names instead of the .java file names